Description: Upstream changes introduced in version 2.7.3-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 libaria (2.7.3-1) unstable; urgency=low
 .
   * Initial release (Closes: #nnnn)  <nnnn is the bug number of your ITP>
 .
 The person named in the Author field signed this changelog entry.
Author: Adam Oleksy <adam.oleksy@pwr.wroc.pl>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ libaria-2.7.3/LICENSE.txt
@@ -0,0 +1,424 @@
+
+  ARIA 
+  MobileRobots Advanced Robotics Interface for Applications
+  
+  License
+
+Note that this license applies to programs which use ARIA
+via both dynamic and static linking or loading, including
+dynamic loading and linking at runtime (e.g. using ArModule,
+Windows LoadLibrary, Unix dlopen, or similar.  In other words
+if you have a plugin which uses ARIA in any way that plugin 
+must also be distributed under this license.)
+
+(Also note that the text of the GPL below is copyrighted by the 
+Free Software Foundation, but that the instance of code that it
+refers to (ARIA) is copyrighted by ActivMedia Robotics 
+LLC and MobileRobots Inc.)
+
+-----------------------------------------------------------------
+
+
+		    GNU GENERAL PUBLIC LICENSE
+		       Version 2, June 1991
+
+ Copyright (C) 1989, 1991 Free Software Foundation, Inc.
+         59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ Everyone is permitted to copy and distribute verbatim copies
+ of this license document, but changing it is not allowed.
+
+			    Preamble
+
+  The licenses for most software are designed to take away
+your freedom to share and change it.  By contrast, the GNU
+General Public License is intended to guarantee your
+freedom to share and change free software--to make sure the
+software is free for all its users.  This General Public
+License applies to most of the Free Software Foundation's
+software and to any other program whose authors commit to
+using it.  (Some other Free Software Foundation software is
+covered by the GNU Library General Public License instead.)  
+You can apply it to your programs, too.
+
+  When we speak of free software, we are referring to
+freedom, not price.  Our General Public Licenses are
+designed to make sure that you have the freedom to
+distribute copies of free software (and charge for this
+service if you wish), that you receive source code or can
+get it if you want it, that you can change the software or
+use pieces of it in new free programs; and that you know
+you can do these things.
+
+  To protect your rights, we need to make restrictions that
+forbid anyone to deny you these rights or to ask you to
+surrender the rights. These restrictions translate to
+certain responsibilities for you if you distribute copies
+of the software, or if you modify it.
+
+  For example, if you distribute copies of such a program,
+whether gratis or for a fee, you must give the recipients
+all the rights that you have.  You must make sure that
+they, too, receive or can get the source code.  And you
+must show them these terms so they know their rights.
+
+  We protect your rights with two steps: (1) copyright the
+software, and (2) offer you this license which gives you
+legal permission to copy, distribute and/or modify the
+software.
+
+  Also, for each author's protection and ours, we want to
+make certain that everyone understands that there is no
+warranty for this free software.  If the software is
+modified by someone else and passed on, we want its
+recipients to know that what they have is not the original,
+so that any problems introduced by others will not reflect
+on the original authors' reputations.
+
+  Finally, any free program is threatened constantly by
+software patents.  We wish to avoid the danger that
+redistributors of a free program will individually obtain
+patent licenses, in effect making the program proprietary.  
+To prevent this, we have made it clear that any patent must
+be licensed for everyone's free use or not licensed at all.
+
+  The precise terms and conditions for copying,
+distribution and modification follow. 
+
+		    GNU GENERAL PUBLIC LICENSE
+TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION AND MODIFICATION
+
+  0. This License applies to any program or other work
+which contains a notice placed by the copyright holder
+saying it may be distributed under the terms of this
+General Public License.  The "Program", below, refers to
+any such program or work, and a "work based on the Program"
+means either the Program or any derivative work under
+copyright law: that is to say, a work containing the
+Program or a portion of it, either verbatim or with
+modifications and/or translated into another language.  
+(Hereinafter, translation is included without limitation in
+the term "modification".)  Each licensee is addressed as
+"you".
+
+Activities other than copying, distribution and
+modification are not covered by this License; they are
+outside its scope.  The act of running the Program is not
+restricted, and the output from the Program is covered only
+if its contents constitute a work based on the Program
+(independent of having been made by running the Program).
+Whether that is true depends on what the Program does.
+
+  1. You may copy and distribute verbatim copies of the
+Program's source code as you receive it, in any medium,
+provided that you conspicuously and appropriately publish
+on each copy an appropriate copyright notice and disclaimer
+of warranty; keep intact all the notices that refer to this
+License and to the absence of any warranty; and give any
+other recipients of the Program a copy of this License
+along with the Program.
+
+You may charge a fee for the physical act of transferring a
+copy, and you may at your option offer warranty protection
+in exchange for a fee.
+
+  2. You may modify your copy or copies of the Program or
+any portion of it, thus forming a work based on the
+Program, and copy and distribute such modifications or work
+under the terms of Section 1 above, provided that you also
+meet all of these conditions:
+
+    a) You must cause the modified files to carry prominent
+    notices stating that you changed the files and the date of
+    any change.
+
+    b) You must cause any work that you distribute or
+    publish, that in whole or in part contains or is derived 
+    from the Program or any part thereof, to be licensed as a 
+    whole at no charge to all third parties under the terms of
+    this License.
+
+    c) If the modified program normally reads commands 
+    interactively  when run, you must cause it, when started 
+    running for such interactive use in the most ordinary way,
+    to print or display an announcement including an 
+    appropriate copyright notice and a notice that there is no
+    warranty (or else, saying that you provide a warranty) and
+    that users may redistribute the program under these 
+    conditions, and telling the user how to view a copy of 
+    this License.  (Exception: if the Program itself is 
+    interactive but does not normally print such an 
+    announcement, your work based on the Program is not
+    required to print an announcement.) These requirements
+apply to the modified work as a whole.  If identifiable
+sections of that work are not derived from the Program, and
+can be reasonably considered independent and separate works
+in themselves, then this License, and its terms, do not
+apply to those sections when you distribute them as
+separate works.  But when you distribute the same sections
+as part of a whole which is a work based on the Program,
+the distribution of the whole must be on the terms of this
+License, whose permissions for other licensees extend to
+the entire whole, and thus to each and every part
+regardless of who wrote it.
+
+Thus, it is not the intent of this section to claim rights
+or contest your rights to work written entirely by you;
+rather, the intent is to exercise the right to control the
+distribution of derivative or collective works based on the
+Program.
+
+In addition, mere aggregation of another work not based on
+the Program with the Program (or with a work based on the
+Program) on a volume of a storage or distribution medium
+does not bring the other work under the scope of this
+License.
+
+  3. You may copy and distribute the Program (or a work
+based on it, under Section 2) in object code or executable
+form under the terms of Sections 1 and 2 above provided
+that you also do one of the following:
+
+    a) Accompany it with the complete corresponding 
+    machine-readable source code, which must be distributed 
+    under the terms of Sections 1 and 2 above on a medium 
+    customarily used for software interchange; or,
+
+    b) Accompany it with a written offer, valid for at least 
+    three years, to give any third party, for a charge no more
+    than your cost of physically performing source
+    distribution, a complete machine-readable copy of the
+    corresponding source code, to be distributed under the
+    terms of Sections 1 and 2 above on a medium customarily
+    used for software interchange; or,
+
+    c) Accompany it with the information you received as to
+    the offer to distribute corresponding source code.  
+    (This alternative is allowed only for noncommercial
+    distribution and only if you received the program in
+    object code or executable form with such an offer, in
+    accord with Subsection b above.)
+
+The source code for a work means the preferred form of the
+work for making modifications to it.  For an executable
+work, complete source code means all the source code for
+all modules it contains, plus any associated interface
+definition files, plus the scripts used to control
+compilation and installation of the executable.  However,
+as a special exception, the source code distributed need
+not include anything that is normally distributed (in
+either source or binary form) with the major components
+(compiler, kernel, and so on) of the operating system on
+which the executable runs, unless that component itself
+accompanies the executable.
+
+If distribution of executable or object code is made by
+offering access to copy from a designated place, then
+offering equivalent access to copy the source code from the
+same place counts as distribution of the source code, even
+though third parties are not compelled to copy the source
+along with the object code. 
+
+
+
+  4. You may not copy, modify, sublicense, or distribute
+the Program except as expressly provided under this
+License.  Any attempt otherwise to copy, modify, sublicense
+or distribute the Program is void, and will automatically
+terminate your rights under this License. However, parties
+who have received copies, or rights, from you under this
+License will not have their licenses terminated so long as
+such parties remain in full compliance.
+
+  5. You are not required to accept this License, since you
+have not signed it.  However, nothing else grants you
+permission to modify or distribute the Program or its
+derivative works.  These actions are prohibited by law if
+you do not accept this License.  Therefore, by modifying or
+distributing the Program (or any work based on the
+Program), you indicate your acceptance of this License to
+do so, and all its terms and conditions for copying,
+distributing or modifying the Program or works based on it.
+
+  6. Each time you redistribute the Program (or any work
+based on the Program), the recipient automatically receives
+a license from the original licensor to copy, distribute or
+modify the Program subject to these terms and conditions.  
+You may not impose any further restrictions on the
+recipients' exercise of the rights granted herein. You are
+not responsible for enforcing compliance by third parties
+to this License.
+
+  7. If, as a consequence of a court judgment or allegation
+of patent infringement or for any other reason (not limited
+to patent issues), conditions are imposed on you (whether
+by court order, agreement or otherwise) that contradict the
+conditions of this License, they do not excuse you from the
+conditions of this License.  If you cannot distribute so as
+to satisfy simultaneously your obligations under this
+License and any other pertinent obligations, then as a
+consequence you may not distribute the Program at all.  
+For example, if a patent license would not permit
+royalty-free redistribution of the Program by all those who
+receive copies directly or indirectly through you, then the
+only way you could satisfy both it and this License would
+be to refrain entirely from distribution of the Program.
+
+If any portion of this section is held invalid or
+unenforceable under any particular circumstance, the
+balance of the section is intended to apply and the section
+as a whole is intended to apply in other circumstances.
+
+It is not the purpose of this section to induce you to
+infringe any patents or other property right claims or to
+contest validity of any such claims; this section has the
+sole purpose of protecting the integrity of the free
+software distribution system, which is implemented by
+public license practices.  Many people have made generous
+contributions to the wide range of software distributed
+through that system in reliance on consistent application
+of that system; it is up to the author/donor to decide if
+he or she is willing to distribute software through any
+other system and a licensee cannot impose that choice.
+
+This section is intended to make thoroughly clear what is
+believed to be a consequence of the rest of this License.
+
+
+
+  8. If the distribution and/or use of the Program is
+restricted in certain countries either by patents or by
+copyrighted interfaces, the original copyright holder who
+places the Program under this License may add an explicit
+geographical distribution limitation excluding those
+countries, so that distribution is permitted only in or
+among countries not thus excluded.  In such case, this
+License incorporates the limitation as if written in the
+body of this License.
+
+  9. The Free Software Foundation may publish revised
+and/or new versions of the General Public License from time
+to time.  Such new versions will be similar in spirit to
+the present version, but may differ in detail to address
+new problems or concerns.
+
+Each version is given a distinguishing version number.  If
+the Program specifies a version number of this License
+which applies to it and "any later version", you have the
+option of following the terms and conditions either of that
+version or of any later version published by the Free
+Software Foundation.  If the Program does not specify a
+version number of this License, you may choose any version
+ever published by the Free Software Foundation.
+
+  10. If you wish to incorporate parts of the Program into
+other free programs whose distribution conditions are
+different, write to the author to ask for permission.  For
+software which is copyrighted by the Free Software
+Foundation, write to the Free Software Foundation; we
+sometimes make exceptions for this.  Our decision will be
+guided by the two goals of preserving the free status of
+all derivatives of our free software and of promoting the
+sharing and reuse of software generally.
+
+			    NO WARRANTY
+
+  11. BECAUSE THE PROGRAM IS LICENSED FREE OF CHARGE, THERE
+IS NO WARRANTY FOR THE PROGRAM, TO THE EXTENT PERMITTED BY
+APPLICABLE LAW.  EXCEPT WHEN OTHERWISE STATED IN WRITING
+THE COPYRIGHT HOLDERS AND/OR OTHER PARTIES PROVIDE THE
+PROGRAM "AS IS" WITHOUT WARRANTY OF ANY KIND, EITHER
+EXPRESSED OR IMPLIED, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A
+PARTICULAR PURPOSE.  THE ENTIRE RISK AS TO THE QUALITY AND
+PERFORMANCE OF THE PROGRAM IS WITH YOU.  SHOULD THE PROGRAM
+PROVE DEFECTIVE, YOU ASSUME THE COST OF ALL NECESSARY
+SERVICING, REPAIR OR CORRECTION.
+
+  12. IN NO EVENT UNLESS REQUIRED BY APPLICABLE LAW OR
+AGREED TO IN WRITING WILL ANY COPYRIGHT HOLDER, OR ANY
+OTHER PARTY WHO MAY MODIFY AND/OR REDISTRIBUTE THE PROGRAM
+AS PERMITTED ABOVE, BE LIABLE TO YOU FOR DAMAGES, INCLUDING
+ANY GENERAL, SPECIAL, INCIDENTAL OR CONSEQUENTIAL DAMAGES
+ARISING OUT OF THE USE OR INABILITY TO USE THE PROGRAM
+(INCLUDING BUT NOT LIMITED TO LOSS OF DATA OR DATA BEING
+RENDERED INACCURATE OR LOSSES SUSTAINED BY YOU OR THIRD
+PARTIES OR A FAILURE OF THE PROGRAM TO OPERATE WITH ANY
+OTHER PROGRAMS), EVEN IF SUCH HOLDER OR OTHER PARTY HAS
+BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES.
+
+		     END OF TERMS AND CONDITIONS
+
+
+
+            How to Apply These Terms to Your New Programs
+
+  If you develop a new program, and you want it to be of
+the greatest possible use to the public, the best way to
+achieve this is to make it free software which everyone can
+redistribute and change under these terms.
+
+  To do so, attach the following notices to the program.  
+It is safest to attach them to the start of each source
+file to most effectively convey the exclusion of warranty;
+and each file should have at least the "copyright" line and
+a pointer to where the full notice is found.
+
+    <one line to give the program's name and a brief idea 
+of what it does.>
+    Copyright (C) <year>  <name of author>
+
+    This program is free software; you can redistribute it
+    and/or modify it under the terms of the GNU General
+    Public License as published by the Free Software
+    Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be
+    useful, but WITHOUT ANY WARRANTY; without even the
+    implied warranty of MERCHANTABILITY or FITNESS FOR A
+    PARTICULAR PURPOSE.  See the GNU General Public License
+    for more details.  You should have received a copy of
+    the GNU General Public License along with this program;
+    if not, write to the Free Software Foundation, Inc., 59
+    Temple Place, Suite 330, Boston, MA 02111-1307 USA
+
+
+Also add information on how to contact you by electronic
+and paper mail.
+
+If the program is interactive, make it output a short
+notice like this when it starts in an interactive mode:
+
+    Gnomovision version 69, Copyright (C) year name of 
+    author Gnomovision comes with ABSOLUTELY NO WARRANTY;
+    for details type `show w'.  This is free software, and
+    you are welcome to redistribute it under certain
+    conditions; type `show c' for details.
+
+The hypothetical commands `show w' and `show c' should show
+the appropriate parts of the General Public License.  Of
+course, the commands you use may be called something other
+than `show w' and `show c'; they could even be mouse-clicks
+or menu items--whatever suits your program.
+
+You should also get your employer (if you work as a
+programmer) or your school, if any, to sign a "copyright
+disclaimer" for the program, if necessary.  Here is a
+sample; alter the names:
+
+  Yoyodyne, Inc., hereby disclaims all copyright interest 
+  in the program 'Gnomovision' (which makes passes at
+  compilers) written by James Hacker.
+
+  <signature of Ty Coon>, 1 April 1989
+  Ty Coon, President of Vice
+
+This General Public License does not permit incorporating
+your program into proprietary programs.  If your program is
+a subroutine library, you may consider it more useful to
+permit linking proprietary applications with the library.  
+If this is what you want to do, use the GNU Library General
+Public License instead of this License.
+
+
+
--- /dev/null
+++ libaria-2.7.3/README.txt
@@ -0,0 +1,424 @@
+
+  ARIA
+  Adept MobileRobots Advanced Robotics Interface for Applications
+
+  Version 2.7.3
+
+Copyright 2002, 2003, 2004, 2005 
+ActivMedia Robotics, LLC. All rights reserved.
+
+Copyright 2006, 2007, 2008, 2009
+MobileRobots Inc. All rights reserved.
+
+Copyright 2010, 2011
+Adept Technology. All rights reserved.
+
+See LICENSE.txt for full license information about ARIA.
+
+Please read this document for important details about using ARIA.
+
+   Contents
+   ========
+*  Introduction
+*  Documentation
+*  Licenses and Sharing
+*  ARIA Package
+*  Files of Note
+*  Compiling programs that use ARIA
+   *  Windows
+   *  Linux
+      *  Using ARIA's Makefiles
+      *  Using your own Makefile or other build system
+      *  Learning more about using Linux
+   *  Learning more about C++
+*  Using ARIA from Python and Java
+*  Simulator
+
+Welcome to the Adept MobileRobots Advanced Robotics Interface for Applications (ARIA).
+ARIA is an object-oriented, application programming interface (API) for
+MobileRobots' (and ActivMedia) line of intelligent mobile robots, including
+Pioneer 2/3 DX and AT, PeopleBot, PowerBot, and AmigoBot mobile robots.
+Written in the C++ language, ARIA is client-side software for easy,
+high-performance access to and management of the robot server, as well
+to the many accessory robot sensors and effectors, and useful utilities. 
+   
+ARIA can be run multi- or single-threaded, using its own wrapper
+around Linux pthreads and WIN32 threads.  Use ARIA in many different
+ways, from simple command-control of the robot server for direct-drive
+navigation, to development of higher-level intelligent actions (aka
+behaviors).   Add your own features and modifications to ARIA: It's 
+open-source!
+
+The ARIA package includes both source code and pre-build libraries
+and example programs. These libraries and programs were build with
+GCC 3.4 if on Linux, and MS Visual Studio .NET 2003 (7.1) if on
+Windows.  Using these compilers for development is recommended. If
+you use a different compiler, you must rebuild the ARIA libraries.
+
+See below for more information about building programs with ARIA
+on Windows and Linux and using the Windows and Linux development tools.
+
+
+Documentation and Help
+======================
+
+Follow the INSTALL text instructions to install ARIA on your Linux or 
+Windows workstation or robot computer. System requirements are included 
+in the INSTALL text.
+
+ARIA includes a full API Reference Manual in HTML format. This manual,
+Aria-Reference.html (and in the docs/ directory), includes documentation 
+about each of the classes and methods in ARIA, as well as a comprehensive 
+overview describing how to get stated understanding and using ARIA.  In 
+addition, ARIA includes several example programs in the examples/ 
+(start with simpleConnect.cpp, then explore the others) and 
+advanced/ directories, and the header files and source code are included
+in include/ and src/ as well.  
+
+The ArNetworking library has its own reference manual,
+ArNetworking-Reference.html in the ArNetworking subdirectory, and examples
+in ArNetworking/examples.
+
+If you plan on using the Java or Python wrapper libraries, see the 
+javaExamples, pythonExamples, ArNetworking/javaExamples and
+ArNetworking/pythonExamples directories for important information in
+README files, and example programs. You should also read the Aria Reference
+manual for general information about Aria -- the API in the wrapper libraries
+are almost identical to the C++ API.
+
+If you have any problems or questions using ARIA or your robot, the 
+MobileRobots support site provides:
+ 
+  * A FAQ (Frequently Asked Questions) list, at <http://robots.mobilerobots.com/FAQ.html>
+  * The Aria-Users mailing list, where you can discuss Aria with other users and 
+    MobileRobots software developers:
+      * Search the archives at <http://robots.mobilerobots.com/archives/aria-users/threads.html>
+      * Join the list at <http://robots.mobilerobots.com/archives/aria-info.html>
+
+
+License and Sharing
+===================
+
+ARIA is released under the GNU Public License, which means that if you
+distribute any work which uses ARIA, you must distribute the entire 
+source code to that work under the GPS as well.  Read the included 
+LICENSE text for details.  We open-sourced ARIA under GPL with full
+source code not only for your convenience, but also so that you can 
+share your enhancements to the software.  If you wish your enhancements 
+to make it into our future ARIA versions, you will need to assign the 
+copyright on those changes to Adept. Contact lafary@mobilerobots.com 
+with these changes or with questions about this.
+
+Accordingly, please do share your work, and please sign up for the 
+exclusive aria-users@mobilerobots.com newslist so that you can benefit 
+from others' work, too.
+
+ARIA may instead be relicensed for proprietary, closed source applications. 
+Contact sales@mobilerobots.com for details.
+
+For answers to frequently asked questions about what the GPL allows
+and requires, see <http://www.gnu.org/licenses/gpl-faq.html>
+
+
+The ARIA Package
+================
+
+Aria/:
+  docs      The API Reference Manual: Extensive documentation of all of ARIA
+  examples  ARIA examples -- a good place to start; see examples/README.txt
+  include   ARIA header (*.h) files
+  src       ARIA source code (*.cpp) files
+  lib       Libraries (.lib files for Windows, .so files for Linux) 
+  bin       Contains Windows binaries and DLLs.
+  params    Robot definition (parameter) files ("p3dx.p", for example)
+  tests     Test files, somewhat esoteric but useful during ARIA development
+  utils     Utility commands, not generally needed
+  advanced  Advanced demos, not for the faint of heart (or ARIA novice)
+  python    Python wrapper package
+  java      Java wrapper package
+
+Aria/ArNetworking/:  (A library used to facilitate network communication)
+  docs      API Reference Manual for ArNetworking 
+  examples  ArNetworking examples
+  include   ArNetworking header (*.h) files, of course
+  src       ArNetworking source (*.cpp) files
+  python    Python wrapper package
+  java      Java wrapper package
+  
+
+Other ARIA Files of Note
+========================
+
+LICENSE.txt   GPL license; agree to this to use ARIA
+Aria.sln      MSVC++ solution for building ARIA and the 'demo' program
+examples/examples.sln MSVC++ solution for building all of the examples
+Makefile      Linux makefile for building ARIA and examples
+Makefile.dep  Linux dependency file
+INSTALL.txt   Step-wise instructions for installing ARIA
+README.txt    This file; also see READMEs in advanced/, examples/, and tests/
+
+
+Compiling a new file(s)
+====================
+
+Windows
+-------
+
+On Windows, use Microsoft Visual Studio .NET 2003 (MS Visual C++ 7.1).
+
+Note: If you use a different version, you must rebuild the ARIA 
+libraries, but MobileRobots only supports Visual Studio .NET 2003,
+and only versions 7.1 and later are probably able to correctly build
+ARIA without adding workarounds to the code.
+
+Open the solution file you want to add your new project to (using
+"File->Open Workspace"), e.g. Aria.sln, or make a new solution 
+(using "File->New->Blank Solution").
+
+Select "File->New->Project", select a console application.  Click the button to
+add to exisiting solution, give it a name. (Delete that name from the 
+'location' field if you want to avoid using a subdirectory.)
+
+Now you must configure your project to find ARIA. These instructions
+assume a default installation; if you installed ARIA elsewhere you
+must use different paths. If you keep your Visual Studo project within
+Aria's directory, you can also use relative paths (e.g. "..\lib" for the
+library path).
+
+1. Create new configuration in the solution: Release and Debug.
+
+2. From the Project Menu, choose to edit your projects Properties. Or
+  right-click on the project in the Solution Explorer and choose Properties.
+
+3. Change the "Configuration" (upper left of the dialog) menu to "All
+  Configurations".
+
+4. Click on the "General" section
+
+  * Set "Output Files" to "C:\Program Files\MobileRobots\Aria\bin" 
+    (Aria's bin directory, where Aria.dll etc. are).  Or, if you want your
+    program saved in a different directory, you must put Aria's bin directory
+    in your system PATH environment variable, or copy the DLLs to your own
+    output directory.
+
+  * You may want to change Intermediate Files. A useful value is
+    "obj\$(ConfigurationName)";  $(ConfigurationName) will be either "Debug" 
+    or "Release".
+
+  * Set "Use Managed Extensions" to "no".
+
+5. Click on the "Link" or "Linker" section.
+
+  * Click on the "Input" subsection.
+
+    * To "Additional Library Path" add "C:\Program Files\MobileRobots\Aria\lib".
+
+    * Change the "Configuration" menu (upper left of the window) to "Debug"
+
+    * If using Visual Studio 2003, To "Additional Dependencies", add:
+        AriaDebug.lib winmm.lib advapi32.lib
+      If you are using the ArNetworking library, also add:
+        ArNetworkingDebug.lib ws2_32.lib
+
+    * Or, if using Visual Studio 2008, To "Additional Dependencies", add:
+        AriaDebugVC9.lib winmm.lib advapi32.lib
+      If you are using the ArNetworking library, also add:
+        ArNetworkingDebugVC9.lib ws2_32.lib
+
+    * Change the "Configuration" menu to "Release"
+
+    * If using Visual Studio 2003, To "Additional Dependencies", add:
+        Aria.lib winmm.lib advapi32.lib
+      If you are using the ArNetworking library, also add:
+        ArNetworking.lib ws2_32.lib
+
+    * Or, if using Visual Studio 2008, To "Additional Dependencies", add:
+        AriaVC9.lib winmm.lib advapi32.lib
+      If you are using the ArNetworking library, also add:
+        ArNetworkingVC9.lib ws2_32.lib
+  
+    * Change the "Configuration" menu back to "All Configurations"
+
+
+6. Click on the "C++" section.
+
+  * Click on the "General" subsection
+
+    * To "Additional Include Directories" add 
+      "C:\Program Files\MobileRobots\Aria\include".
+
+   * Click on the "Code Generation" subsection
+
+     * Change the "Configuration" menu (upper left of the window) to "Debug".
+
+       * Under the "Use run-time library" pulldown select "Debug
+         Multithreaded DLL".
+
+     * Change the "Configuration" menu (upper left of the screen) to "Release".
+
+       * Under the "Use run-time library" pulldown select "Multithreaded DLL".
+
+     * Change the "Configuration" menu back to "All Configurations"
+
+Linux
+-----
+
+On GNU/Linux two tools are used: the compiler (GCC), which compiles
+and/or links a single file, and Make, which is used to manage 
+building multiple files and their dependencies.  If you need to
+deal with multiple files, you can copy a Makefile and modify it, 
+or create a new one.
+
+Note: In all packages for RedHat and Debian, the shared libraries
+were built with G++ 3.4. You must use G++ 3.4 or a compatible version to
+build programs that link against it, or rebuild ARIA using your 
+preferrend G++ version. G++ 3.4 has been preinstalled on robot on-board 
+computers with RedHat 7.3 in /usr/local/gcc-3.4.1, and it has been configured
+as the default gcc and g++ compilers on Debian systems as well. (On other
+Debian systems, the default g++ and gcc compiler versions may be changed
+using the 'galternatives' program or the 'update-alternatives' command). 
+You may need to install it yourself on your own system if not already present. 
+When compiling ARIA or its examples, you may also temporarily override the 
+C++ compiler command ('g++' by default) by setting the CXX environment 
+variable before or while running make. E.g.: "make CXX=g++-3.4" or 
+"export CXX=g++-3.4; make".
+
+More information and documentation about GCC is available at 
+<http://gcc.gnu.org>, and in the gcc 'man' page.  More information 
+about Make is at <http://www.gnu.org/software/make/>. Several graphical 
+IDE applications are available for use with GCC, though none are 
+currently supported by MobileRobots.
+
+
+
+# Using ARIA's Makefiles on Linux: #
+
+The ARIA Makefile is able to build any program consisting 
+of one source code file in the 'examples', 'tests', or 'advanced' 
+directory.  For example, if your source code file is 
+'examples/newProgram.cpp', then you can run 'make examples/newProgram' 
+from the top-level Aria directory, or 'make newProgram' from within 
+the 'examples' directory, to build it.   This makes it easy to copy
+one of the example programs to a new name and modify it -- a good way
+to learn how to use ARIA.
+
+
+
+# Using ARIA from your own Makefile or other build system: #
+
+If you want to keep your program in a different place than the Aria 
+examples directory, and use your own Makefile or other build tool,
+you need to use the following g++ options to compile the program:
+
+ -fPIC -I/usr/local/Aria/include 
+
+If you wish, you may also use -g and -Wall for debugging information
+and useful warnings.   Aria does not use exceptions, so you may also
+use -fno-exceptions if you wish; this will cause any use of exceptions
+in your program to trigger a compile error.
+
+For linking with libAria use these options:
+
+  -L/usr/local/Aria/lib -lAria -lpthread -ldl -lrt
+
+If you are also using ArNetworking, use the following compile option
+in addition to the Aria options above:
+
+  -I/usr/local/Aria/ArNetworking/include
+  
+And for linking to libArNetworking:
+
+  -lArNetworking
+
+
+A Makefile for a program called "program" with source file "program.cpp"
+might look something like this:
+
+
+all: program
+
+CFLAGS=-fPIC -g -Wall
+ARIA_INCLUDE=-I/usr/local/Aria/include
+ARIA_LINK=-L/usr/local/Aria/lib -lAria -lpthread -ldl
+
+%: %.cpp
+	$(CXX) $(CFLAGS) $(ARIA_INCLUDE) $< -o $@ $(ARIA_LINK)
+
+
+Refer to the GNU Make manual <http://www.gnu.org/software/make> or 
+other books or documentation about Make to learn more.
+
+
+# Learning more about using Linux: #
+
+If you are new to using GNU/Linux, some guides on getting started can be 
+found at the following sites:
+
+ * If your robot is using RedHat (it displays "RedHat Linux" on the console), 
+   start with
+   <https://www.redhat.com/docs/manuals/linux/RHL-7.3-Manual/getting-started-guide/>.
+   More at <https://www.redhat.com/support/resources/howto/rhl73.html>.
+
+ * If your robot is using Debian, start with 
+   <http://www.us.debian.org/doc/manuals/debian-tutorial>. More is at
+   <http://www.debian.org/doc/>.
+
+ * <http://www.tldp.org> is a repository of many different guides, FAQ and
+   HOWTO documents about Linux in general, including various programming
+   and system administration tasks.
+
+For more depth, there are many books about using Linux, consult your
+local computer bookseller. The ideal way to learn about Linux is to work 
+with an experienced colleague who can demonstrate things and answer 
+questions as they arise.
+
+
+
+Learning C++
+------------
+
+If you are new to C++ programming, the definitive guide is Bjarne
+Stroustrup's book "The C++ Programming Language".   The book
+"C++ In a Nutshell", published by O'Reilly & Associates, is a
+good quick guide and reference. There are also several websites
+and many other books about C++.
+
+
+Using ARIA from Java or Python
+==============================
+
+Even though ARIA was written in C++, it is possible to use ARIA from
+the Java and Python programming languages as well. The directories
+'python' and 'java' contain Python and Java packages automatically generated
+by the 'swig' tool (http://www.swig.org) which mirror the ARIA API
+closely, and are "wrappers" around the native ARIA library: When you call
+an ARIA function in a Python or Java program, the wrapper re-marshalls the
+function arguments and makes the equivalent call into the C++ ARIA library.
+See the 'pythonExamples' and 'javaExamples' directories for more information
+and example programs.  See the ARIA API Reference Manual in 'docs' for 
+some more information.
+
+More about the Python programming language is at <http://www.python.org>.
+More about the Java programming language is at <http://java.sun.com>.
+
+
+MobileSim Simulator
+===================
+
+SRIsim is no longer included with Aria.  There is now a seperately
+downloadable MobileSim simulator at our support webpage 
+<http://robots.mobilerobots.com>.  MobileSim is compatible with SRISim,
+but adds many new features.
+
+
+
+
+-------------------------
+Adept MobileRobots
+10 Columbia Drive
+Amherst, NH, 03031. USA
+
+support@mobilerobots.com
+http://www.activrobots.com
+
+
--- libaria-2.7.3.orig/aria/doc/CMakeLists.txt
+++ libaria-2.7.3/aria/doc/CMakeLists.txt
@@ -26,7 +26,7 @@ if (ARIA_BUILD_DOCUMENTATION)
     add_dependencies (doc ariaShared)
     
     # Install documentation
-    set (docInstallDir "share/doc/aria")
+    set (docInstallDir "share/doc/libaria")
     set (docSourceDir "${CMAKE_CURRENT_BINARY_DIR}/html")
     # Make the output directory now so install doesn't complain that it doesn't exist
     file (MAKE_DIRECTORY ${docSourceDir})
